vectorized2d/vector2d.py

```diff
@@ -84,6 +84,26 @@ def rotated(self, rotation_angle: Union[float, np.ndarray, Iterable[float]],
         new_direction = self.direction + rotation_angle
         return Vector2D(magnitude=self.norm, direction=new_direction)
 
+    @staticmethod
+    @njit
+    def _calc_angle_diff(direction_from: np.ndarray, direction_to: np.ndarray) -> np.ndarray:
+        raw_diff = direction_to - direction_from
+        diff = np.abs(raw_diff) % (2 * np.pi)
+        diff[diff > np.pi] = 2 * np.pi - diff[diff > np.pi]
+        signs = np.full_like(diff, -1)
+        signs[np.logical_or(
```

Comment:
Have you tried comparing this **time-wise** to the corresponding bit-wise operations (`&, |`)?